Update Get-Team.md (#7477)
MicrosoftDocs/office-docs-powershell: teams/teams-ps/teams/Get-Team.md

This cmdlet supports retrieving teams with particular properties/information, including all teams that a specific user belongs to, all teams that have been archived, all teams with a specific display name, or all teams in the organization.

> >[!NOTE]

Reviewer: ```suggestion
> [!NOTE]
```

> >At times, Get-Team returns multiple results matching the input and not just the exact match for attributes like DisplayName/MailNickName. This is a known behaviour.

Reviewer: ```suggestion
> Get-Team may return multiple results matching the input and not just the exact match for attributes like DisplayName/MailNickName. This is known behavior.
```
